index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="./styles/style.css">
+    <title>VueJs | Amsterdam</title>
+</head>
+<body>
+    <nav class="nav-header">
+        <i class="fa fa-facebook nav-icons" aria-hidden="true"></i>
+        <i class="fa fa-twitter nav-icons" aria-hidden="true"></i>
+        <ul class="nav-header-list">
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+    </nav>
+    <header class="header">
+        <img src="./images/logs.png" alt="" class="logo">
+        <ul class="nav-header-list">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="./index.html">Program</a></li>
+            <li><a href="./index.html">Join</a></li>
+            <li><a href="./index.html">Sponsor</a></li>
+            <li><a href="./index.html">News</a></li>
+            <li><button class="btn-header">Join us in Amsterdam</button></li>
+        </ul>
+    </header>
+
+    <header class="h-desktop" id='menu'>
+        <i class="fa fa-times fa-2x" id='close-btn' aria-hidden="true"></i>
+        <ul class="nav-element">
+            <li><a class="nav-link" href="./about.html">About</a></li>
+            <li><a class="nav-link" href="#">Program</a></li>
+            <li><a class="nav-link" href="#">News</a></li>
+        </ul>
+    
+    </header>
+      
+    <section class="mob-nav-bar" >
+        <i class="fa fa-bars fa-1x"  id="menu-btn" aria-hidden="true"></i>
+        
+    </section>
+    <main>
+        <!-- intro -->
+    <section class="intro">
+        <div class="intro-content">
+            <h1>"VueJs Amsterdam"</h1>
+            <p class="intro-title">
+                WORLD'S MOST<br>
+                SPECIAL AND<br>
+                LARGEST VUE CONFERENCE
+            </p>
+            <section class="intro-desc">
+                Attend the Vue.js Community Event of the year and celebrate with Vue Communities
+            </section>
+            <p class="intro-date">2022.6.2(MON)~3(TUE)</p>
+            <span class="intro-location">In Amsterdam</span>
+        </div>
```

Comment:
- [ ] Hi the text with the date in this section should be medium-sized. It should not be as big as the main text title (in orange). please reduce the size of the text that shows the date to something slightly smaller than the main title text.
- [ ] You are currently not using any of the fonts used in the reference design. Please use the correct fonts used in the reference design in this section and also on all other sections. You should only have two fonts on the page; `COCOGOOSE` and `Lato`. please use only these two fonts on the page. 
- [ ] Fix the fonts in all other places on the website as well.

Reply:
# As suggested
<img width="1440" alt="Screen Shot 2022-02-04 at 17 15 16" src="https://user-images.githubusercontent.com/40362530/152553768-ce0430ec-913f-4737-8cda-87466959135b.png">